.ocamlformat

```diff
@@ -0,0 +1,2 @@
+break-cases=toplevel
```

Comment:
Has everybody agreed to use ocamlformat?

Reply:
No, we have not agreed to use ocamlformat.

Reply:
...but we are holding a developers meeting in just under an hour in which we will discuss if / how we might use ocamlformat

Reply:
No there has not been consensus for ocamlformat, but there is a current approach to do a per file consistency and these are the settings I use for consistency within the lens files. Unfortunately the `.ocamlformat` file seems to not work on a per library but a per project workspace so it would need to be in the parent folder. `ocamlformat` is not run automatically or checked though.

Reply:
I think this is junk that shouldn't be added to revision control, unless we agree to use `ocamlformat` project-wide.

Reply:
As mentioned in#504, yesterday we agreed on meeting in an opt-in, noninvasive version of the ocamlformat setup Rudi is using so that others can try it and we can make an informed decision whether to use it more systematically.  

If more than one person wants to use it then I think we should have a single common set of settings.  If not then we can eventually delete again.

This file can be removed from this PR but I guess a similar one is or will be in #504.